### PR TITLE
move linters to guides [Delivers #50hc697n]

### PR DESCRIPTION
**Pull request by balvig**

### CODE REVIEW

File: linters/rubocop.yml [hunk 1]
```diff
@@ -0,0 +1,313 @@
+AllCops:
+  TargetRubyVersion: 2.3
+  Exclude:
+    - config/unicorn/**/*
+    - db/schema.rb
+    - db_chat/schema.rb
+    - vendor/**/*
+    - lib/tasks/dodocook/**/*
```
Comment (Knack) [inline linters/rubocop.yml]: We should move this line (and `db_chat/schema.rb` and maybe the `unicorn` line) to the internal `.rubocop.yml` in our global-web project.

File: linters/README.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+## Usage
+
+Add to the following files:
+
+#### .rubocop.yml
+
+```yml
+inherit_from:
+  - https://raw.githubusercontent.com/cookpad/guides/master/linters/rubocop.yml
+```
```
Comment (Knack) [inline linters/README.md]: I would add:

---

#### .gitignore

Ignore `.rubocop-http*` files which are cached files.

```
.rubocop-http*
```

File: linters/rubocop.yml (hunk 1 above)
Comment (balvig) [inline linters/rubocop.yml]: Good point 👍

File: linters/README.md (hunk 2 above)
Comment (balvig) [inline linters/README.md]: Ah, maybe this belongs in your global gitconfig instead? I don't see those files personally (could be IDE related).

File: linters/rubocop.yml (hunk 1 above)
Comment (balvig) [inline linters/rubocop.yml]: https://github.com/cookpad/guides/pull/40/commits/952ea91fd3a100485670ecfaa905dfd2e5f8b2e1

Comment (Knack) [inline linters/rubocop.yml]: 🤔 Maybe the whole `AllCops` block should be specified in each project? We are assuming here a specific Ruby version and a project structure.

Comment (Knack) [inline linters/rubocop.yml]: See this repo as an example: https://github.com/pixta-dev/pixta-rubocop

Comment (balvig) [inline linters/rubocop.yml]: Also a good point 👍  😅

Comment (balvig) [inline linters/rubocop.yml]: https://github.com/cookpad/guides/pull/40/commits/3e5607cec401473f71aaad5fe10f7866ba48005d

File: linters/README.md (hunk 2 above)
Comment (Knack) [inline linters/README.md]: Maybe it's related to _your_ IDE :-). I see it as a untracked file when running `git status` from the command line. I guess every developer will have this issue, so it's ok to write some documentation explaining it. They can later decide where to ignore it.

Comment (balvig) [inline linters/README.md]: > I guess every developer will have this issue

lol, what I mean is, this file does not get generated on my system (so in other words not an issue for _every_ developer 😉 ). 

I tend to add things that I need in every project to my own `.gitignore_global` file to not have to add it every time. Especially if it is environment-dependent (ie `.DS_Store` etc).

Comment (Knack) [inline linters/README.md]: > maybe I need to do the external file thing first though
🙄

**Pull request merged**